src/module-vsbridge-indexer-cms/Model/Indexer/CmsBlock.php

```diff
@@ -59,6 +68,7 @@ public function execute($ids)
         foreach ($stores as $store) {
             $this->indexHandler->saveIndex($this->cmsBlockAction->rebuild($store->getId(), $ids), $store);
             $this->indexHandler->cleanUpByTransactionKey($store, $ids);
+            $this->cacheProcessor->cleanCacheByTags($store->getId(), ['cmsBlock']);
```

Comment:
I would use 
` $this->cacheProcessor->cleanCacheByDocIds($storeId, $this->indexHandler->getIndexIdentifier(), $ids);`
and below:
` $this->cacheProcessor->cleanCacheByTags($store->getId(), [$this->indexHandler->getIndexIdentifier()]);`


instead, so we can have tags same as type/identifier in the configuration: "cms_block" or "cms_page"
What do you think?
Is VSF core using these tags?

Reply:
That would require refactoring of the VSF code. because the caching tag there is not with a underscore.

Reply:
OK, I got it. Thanks. I will merge this as it is and think later about a different solution.
Right now, even you are reindexing data partially (`execute($ids)`), you still flush cache for all cms content (pare or block).